### ISSUE TITLE
Vulkan support?

### ISSUE THREAD

**Issue by oscarbg** [GameTechDev/IntelShaderAnalyzer#1]
just curious to inspect ISA also for SPIR-V shaders.. tool could also support GLSL input (Vulkan variant) via glslang..

**Comment by oscarbg** [comment]
forget there is also no way to get ISA from OpenGL shaders too.. Hope there is a IntelShaderAnalyzer 2.0 release with Vulkan+OpenGL support too..

**Comment by JeffersonMontgomery-Intel** [comment]
Yes, SPIR-V input is not currently supported but we do plan to add that in a future release.

**Comment by oscarbg** [comment]
thanks.. that’s good news..

**Comment by Valkirie** [comment]
@JeffersonMontgomery-Intel any update ?

**Comment by DaveBookout-Intel** [comment]
sorry, we do not plan to support this.

**Comment by oscarbg** [comment]
@DaveBookout-Intel OK thanks, sad news.. does this indirectly mean that Intel GPA product will not support a Vulkan Shader Profiler showing ISA for a long time, right? (https://www.intel.com/content/www/us/en/develop/documentation/gpa-user-guide/top/analyze-desktop-apps/optimize-specific-domains/optimize-shaders.html)

**Comment by DaveBookout-Intel** [comment]
No, this is specific to the API that this tools is using. I'm sorry, I don't know the history of this project to know what the original plans were or why those plans changed. This should not impact GPA, but let me double check that :)
